op-chain-ops/upgrades/l1.go

```diff
@@ -420,7 +420,7 @@ func SystemConfig(batch *safe.Batch, implementations superchain.ImplementationLi
 	}
 
 	calldata, err := initialize.Inputs.PackValues([]any{
-		finalSystemOwner,
```

Comment:
The base that this is implemented on has bugs in this file, should be a simple rebase when its ready

Reply:
I'd like to do a rebase ASAP, appreciate the heads up.